backend/fms_core/models/dataset.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.core.exceptions import ValidationError
+import reversion
+
+from .tracked_model import TrackedModel
+from .project import Project
+from .experiment_run import ExperimentRun
+from .container import Container
+
+from ._constants import STANDARD_NAME_FIELD_LENGTH
+
+@reversion.register()
+class Dataset(TrackedModel):
+    """ Class to store information about the datasets of data deliveries. """
+
+    project_name = models.CharField(max_length=STANDARD_NAME_FIELD_LENGTH, help_text="Project name.")
+
+    run_name = models.CharField(max_length=STANDARD_NAME_FIELD_LENGTH, help_text="Run name.")
+
+    lane = models.CharField(max_length=10, blank=True, help_text="Coordinates of the lane in a container")
+
+    def save(self, *args, **kwargs):
+        errors = {}
+
+        try:
+            int(self.lane)
+        except ValueError as e:
+            errors["ValueError"] = "The lane must be a positive integer"
```

Comment:
you do not test for negative ... using -5 will pass.

Reply:
Do we allow lane number 0? Just in case they are programmers?